alexa-remote.js

```diff
@@ -2910,7 +2910,7 @@ class AlexaRemote extends EventEmitter {
         const operationPayload = {
             'deviceType': dev.deviceType,
             'deviceSerialNumber': dev.serialNumber,
-            'locale': 'de', // TODO!!
+            'locale': this._options.acceptLanguage, // TODO!!
```

Comment:
Did you tested that this works becazsuse here no locale is used but "just" a language code ... ?

Reply:
Ok I wait till you have tested otr I found time

Reply:
I tested with fr and de, no change... Alexa speaks in French (because configuration in fr for my tests)